bytecomp/semantics_of_primitives.ml

```diff
@@ -142,14 +142,17 @@ let for_primitive (prim : Lambda.primitive) =
   | Pint_as_pointer -> No_effects, No_coeffects
   | Popaque -> Arbitrary_effects, Has_coeffects
   | Ploc _ ->
-    Misc.fatal_error "[Ploc] should have been eliminated by [Translcore]"
+      (* Removed by [Translcore]. *)
+      No_effects, No_coeffects
   | Prevapply
-  | Pdirapply
+  | Pdirapply ->
```

Comment:
I will manage (no doubt it will cause a merge conflict somewhere though, but not to worry)